frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -207,11 +207,11 @@ The alternative status bar can be configured here.]])
 
 function ReaderCoptListener:getAltStatusBarMenu()
     return {
-        text = _("Alt status bar"),
+        text = _("Alternative status bar"),
         separator = true,
         sub_item_table = {
             {
-                text = _("About alternate status bar"),
+                text = _("About alternative status bar"),
```

Comment:
not a fan on this one. I would prefeer "Alt status bar", if "alternate" isn't good enough.

Reply:
A straggler
```suggestion
                text = _("About alt status bar"),
```